decred/decred/dcr/account.py

```diff
@@ -1907,13 +1907,13 @@ def purchaseTickets(self, qty, price):
             spendLimit=int(round(price * qty * 1.1 * 1e8)),  # convert to atoms here
             poolAddress=pi.poolAddress,
             votingAddress=pi.ticketAddress,
-            ticketFee=0,  # use network default
+            ticketFee=DefaultRelayFeePerKb,
```

Comment:
I noticed this is set separately in dcrwallet with `setticketfee`. We could 1) Leave it like it is. All tickets at the network default 10 atoms/byte rate, or 2) Use the user's `relayFee` setting, or 3) implement our own `setTicketFee`.

Reply:
When trying to use our relay fee, default and high were fine, but if I tried to use the lower 4 atoms/b setting, dcrdata threw an error back at me. So, either default or the `setticketfee` you mention.  Is default ok for this pr, since that wouldn't change anything, and we can investigate `setticketfee` in another issue/pr?